packages/wasm-dpp/lib/test/fixtures/getDocumentsFixture.js

```diff
@@ -0,0 +1,43 @@
+const crypto = require('crypto');
+
+const getDataContractFixture = require('./getDataContractFixture');
+
+const { default: loadWasmDpp } = require('../../..');
+let { DocumentFactory, DocumentValidator, ProtocolVersionValidator } = require('../../..');
+const generateRandomIdentifierAsync = require('../utils/generateRandomIdentifierAsync');
+
+/**
+ * @param {DataContract} [dataContract]
+ * @return {Promise<Document[]>}
+ */
+module.exports = async function getDocumentsFixture(
+  dataContract,
+) {
+  if (!dataContract) {
+    // eslint-disable-next-line no-param-reassign
+    dataContract = await getDataContractFixture();
+  }
+
+  ({ DocumentFactory, DocumentValidator, ProtocolVersionValidator } = await loadWasmDpp());
+
+  const documentValidator = new DocumentValidator(new ProtocolVersionValidator());
+  const factory = new DocumentFactory(1, documentValidator, {});
+
+  const ownerId = await generateRandomIdentifierAsync();
+
+  return [
+    factory.create(dataContract, ownerId, 'niceDocument', { name: 'Cutie' }),
+    factory.create(dataContract, ownerId, 'prettyDocument', { lastName: 'Shiny' }),
+    factory.create(dataContract, ownerId, 'prettyDocument', { lastName: 'Sweety' }),
+    factory.create(dataContract, ownerId, 'indexedDocument', { firstName: 'William', lastName: 'Birkin' }),
+    factory.create(dataContract, ownerId, 'indexedDocument', { firstName: 'Leon', lastName: 'Kennedy' }),
+    factory.create(dataContract, ownerId, 'noTimeDocument', { name: 'ImOutOfTime' }),
+    factory.create(dataContract, ownerId, 'uniqueDates', { firstName: 'John' }),
+    factory.create(dataContract, ownerId, 'indexedDocument', { firstName: 'Bill', lastName: 'Gates' }),
+    factory.create(dataContract, ownerId, 'withByteArrays', {
+      byteArrayField: crypto.randomBytes(10),
+      identifierField: (await generateRandomIdentifierAsync()).toBuffer(),
```

Comment:
Please check this one. 
Even though we have `application/x.dash.dpp.identifier` as a contentMediaType for this field in Data Contract fixture, it does not accept wasm Identifier, but only buffer. I wasn't sure if it's a bug or not.

cc: @qrayven

Reply:
They should be, but I think we've decided to work with `Buffer` only here for now because accepting `Identifier` as well would be a bit of a hassle. But I agree, it should take both